src/Helper/HeadLink.php

```diff
@@ -290,15 +288,19 @@ public function set($value)
             );
         }
 
-        return $this->getContainer()->set($value);
+        $this->getContainer()->set($value);
+
+        return $this;
     }
 
     /**
      * Create HTML link element from data item
      *
+     * @internal This method will become private in version 3.0
+     *
      * @return string
      */
-    public function itemToString(stdClass $item)
+    public function itemToString(object $item)
```

Comment:
If I test this locally, it ends in a BC break.

Reply:
Yes. My bad… I'm getting mixed up with covariance and contravariance. This is a BC break for inheritors - I'll revert parameter type changes throughout 👍